studio/schemas/documents/videoResource.js

```diff
@@ -37,6 +37,28 @@ export default {
           return true
         }),
     },
+    {
```

Comment:
The fields we've added to the `videoResource` document used to be attributes associated with a lesson. This was a chance to improve the data model by moving things like duration, transcript, and subtitles to the video resource. This makes the lesson document more flexible, it could be associated with any type of primitive resource, not just a video.

Reply:
lesson has been so overloaded forever, this is great